src/Screens/Main/Chat/index.tsx

```diff
@@ -19,11 +19,17 @@ const Chat = ({ navigation }: Props) => {
     navigation.goBack();
   };
   const buddyId = navigation.getParam('buddyId');
+  const isBanned = navigation.getParam('isBanned');
   const keyboardViewBehaviour =
     RN.Platform.OS === 'ios' ? 'padding' : undefined;
   return (
     <RN.View style={styles.screen}>
-      <Title style={styles.title} onPress={goBack} buddyId={buddyId} />
+      <Title
+        style={styles.title}
+        onPress={goBack}
+        buddyId={buddyId}
+        isBanned={isBanned}
```

Comment:
This "isBanned" should not be propped but calculated from state with buddyId using selector.